docs/architecture.md

```diff
@@ -32,3 +32,11 @@ related:
 * {{ "lobby" | dfn }}
 * {{ "reception" | dfn }}
 * {{ "room" | dfn }}
+
+## Stages
+
+* rough-in or cut-out
```

Comment:
There appear to be definitions on Wordnik without the hyphens, should we use them instead?

```suggestion
* {{ "rough in" | dfn }} or {{ "cut-out" | dfn }}
```